scaler/handlers_test.go

```diff
@@ -185,140 +234,8 @@ func TestStreamIsActive(t *testing.T) {
 				t.Fatalf("expected no error but got: %v", err)
 			}
 
-			if tc.expected != res.Result {
-				t.Fatalf("Expected IsActive result %v, got: %v", tc.expected, res.Result)
-			}
-		})
-	}
-}
-
-func TestIsActive(t *testing.T) {
```

Comment:
This test was duplicated. Tests cases were almost the same as for IsActiveStream and inside IsActiveStream we call to IsActive